README.md

```diff
@@ -2,78 +2,106 @@
 
-This project does not support the same set of Bidders as Prebid.js, although there is overlap.
-The current set can be found in the [adapters](./adapters) package. If you don't see the one you want, feel free to [contribute it](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html).
+## Getting Started
+- [What is Prebid Server](https://docs.prebid.org/prebid-server/overview/prebid-server-overview.html)
+- [Intro to Header Bidding](https://docs.prebid.org/overview/intro-to-header-bidding.html)
+- [Header Bidding with Prebid](https://docs.prebid.org/overview/intro.html#header-bidding-with-prebid)
+- [API Endpoints](https://docs.prebid.org/prebid-server/endpoints/pbs-endpoint-overview.html)
+  
+## Configuring
 
-For more information, see:
+When hosting Prebid Server or developing locally, **you must set a default GDPR value**. This configuration determines whether GDPR is enabled when no regulatory signal is available in the request, where a value of `"0"` disables it by default and a value of `"1"` enables it. This is required as there is no consensus on a good default.
 
-- [What is Prebid?](https://docs.prebid.org/overview/intro.html)
-- [Prebid Server Overview](https://docs.prebid.org/prebid-server/overview/prebid-server-overview.html)
-- [Current Bidders](https://docs.prebid.org/dev-docs/pbs-bidders.html)
+Refer to the [configuration guide](docs/developers/configuration.md) for additional information and a list of available configuration options.
 
-Please consider [registering your Prebid Server](https://docs.prebid.org/prebid-server/hosting/pbs-hosting.html#optional-registration) to get on the mailing list for updates, etc.
+## Hosting Prebid Server
+> [!NOTE]
+> Please consider [registering as a Prebid Server host](https://docs.prebid.org/prebid-server/hosting/pbs-hosting.html#optional-registration) to join the mailing list for updates and feedback.
 
-## Installation
+The quickest way to host Prebid Server is to deploy our [official Docker image](https://hub.docker.com/r/prebid/prebid-server). If you're hosting the container with Kubernetes, you can configure Prebid Server with environment variables [using a pod file](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/) or [using a config map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables). Alternatively, you can use a configuration file [embedded in a config map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap) which Prebid Server will read from at the path `/etc/config`.
 
-First install [Go](https://golang.org/doc/install) version 1.19 or newer.
+For deploying a fork, you can create a custom Docker container using the command:
+``` bash
+docker build -t prebid-server .
```

Comment:
@SyntaxNode I would suggest adding a note before we refactor the image that is also ARM friendly.

so I would suggest explicitly adding `--platform linux/amd64`

Reply:
Circling back to this after the winter break. Is the use case building a deployment container on an Apple processor MacBook?